deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs.yaml

```diff
@@ -114,6 +114,10 @@ spec:
               globalCIDR:
                 description: GlobalCIDR specifies the global CIDR used by the cluster.
                 type: string
+              ignoreCertificateErrors:
```

Comment:
I’m not sure how easy it is to override a default-`true` boolean (I fear a `false` setting might get lost somewhere because of `omitempty`); I need to run a few experiments.

Reply:
See https://github.com/stolostron/submariner-addon/pull/1188 for an alternate proposal.